packages/effect/src/Schema.ts

```diff
@@ -7174,7 +7174,8 @@ const bigDecimalPretty = (): pretty_.Pretty<bigDecimal_.BigDecimal> => (val) =>
   `BigDecimal(${bigDecimal_.format(bigDecimal_.normalize(val))})`
 
 const bigDecimalArbitrary = (): LazyArbitrary<bigDecimal_.BigDecimal> => (fc) =>
-  fc.tuple(fc.bigInt(), fc.integer()).map(([value, scale]) => bigDecimal_.make(value, scale))
+  fc.tuple(fc.bigInt(), fc.integer({ min: 0, max: 18 }))
+    .map(([value, scale]) => bigDecimal_.make(value, scale))
```

Comment:
Currently (both before and after the proposed change), this can generate non-normalized BigDecimal's of the form:

```ts
BigDecimal.make(1234000n, 0) // Would normalize to BigDecimal.make(1234n, -4)
```

I'm wondering if we should normalize the generated values.

```ts
BigDecimal.normalize(BigDecimal.make(value, scale))
```

Reply:
It's not a big deal becaues the module still renders / formats them correctly and also calculates correctly with them (of course). Normalization just matters for the internal representation and if you print the raw properties of the object.